setup.py

```diff
@@ -16,7 +16,9 @@
     packages=setuptools.find_packages(),
     install_requires=[
         "requests",
-        "iso8601"
+        "iso8601",
+        "pytz",
+        "tzinfo"
```

Comment:
`tzinfo` seems to not exist as an independent package, probably included in pytz

Reply:
I think you mean `tzlocal` (the package) here?

Reply:
possible, build seems to work now